spec/acceptance/async_http_client/async_http_client_spec.rb

```diff
@@ -362,6 +362,8 @@ def make_request(method, url, protocol: nil, headers: {}, body: nil)
           e
         end
       end.wait
+
+      result.is_a?(Exception) ? raise(result) : result
```

Comment:
🗒️ Before async 2.6.4, `wait` would raise an exception if the task returned an exception object (in this case, `Async::TimeoutError`). Since 2.6.4, `wait` no longer raises the exception, it just returns it as an exception object.

The webmock specs rely on the exception being raised, so I added this ternary line to restore the old `wait` behavior.